wallet/createtx.go

```diff
@@ -179,6 +180,10 @@ func (w *Wallet) txToOutputs(outputs []*wire.TxOut,
 
 		var inputSource txauthor.InputSource
 		if len(selectedUtxos) > 0 {
+			if !fn.IsUniqueVals(selectedUtxos) {
```

Comment:
I think it makes sense to just inline this function here, since it's literally a single line, instead of waiting for another PR to be merged and tag to be pushed.